### DISCUSSION TITLE
Why nesting `props` when returning from `getStaticProps` or `getServerSideProps`?

### DISCUSSION THREAD

**Discussion by tomdohnal** [vercel/next.js#11294] [Help] [answered]
When using the `getInitialProps` method, we return the `props` directly like this
```
return { data: myData }
```

This is not the case with `getStaticProps` and `getServerSideProps` when we nest it like this:
```
return { props: { data: myData } }
```

I wonder what the reasons are for introducing a more verbose syntax :)

**Comment by timneutkens** [comment, marked as answer]
Additional options will be introduced in the future.

**Reply by tomdohnal** [reply]
Thanks :)
